documenters_aggregator/spiders/accc.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+import scrapy
+from datetime import datetime
+from dateutil.parser import parse as dateparse
+
+
+class AcccSpider(scrapy.Spider):
+    name = 'accc'
+    long_name = 'Animal Care and Control Commission'
+    allowed_domains = ['www.cityofchicago.org']
+    start_urls = ['https://www.cityofchicago.org/city/en/depts/cacc/supp_info/public_notice.html']
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css('.page-full-description').css("ul").css("li"):
+            # Pull the string
+            try:
+                text = item.xpath("text()").extract()[0]
+            except IndexError:
+                continue
+
+            # Strip it
+            text = text.strip()
+
+            # Pass if there's nothing left
+            if not text:
+                continue
+
+            # Parse the item
+            yield {
+                '_type': 'event',
+                'id': self._parse_id(text),
+                'name': self._parse_name(text),
+                'description': self._parse_description(text),
+                'classification': self._parse_classification(text),
+                'start_time': self._parse_start(text),
+                'end_time': self._parse_end(text),
+                'all_day': self._parse_all_day(text),
+                'status': self._parse_status(text),
+                'location': self._parse_location(text),
+            }
+
+    def _parse_id(self, item):
+        """
+        Calulate ID. ID must be unique within the data source being scraped.
+        """
+        return None
```

Comment:
We need an ID for the event. It just has to be unique within the universe of data the spider is collecting, so you could just slugify name and date/time as long as they never have more than one meeting at the same time. Or see if there's an ID in the DOM.